<?xml version="1.0"?>
<launch>
  <!-- dummy scan -->
  <arg name="scan_topic" default="map"/>
  <node name="dummy_scan" pkg="seed_r7_navigation" type="dummy_scan.py">
    <remap from="map" to="$(arg scan_topic)"/>
  </node>

  <!-- Run map navigation -->
  <arg name="map_localization_file"    default="$(find seed_r7_navigation)/maps/dummy.yaml" />
  <arg name="map_keepout_file"         default="$(arg map_localization_file)"  />

  <!-- Run map navigation -->
  <arg name="cmd_vel_topic" default="cmd_vel"/>
  <include file="$(find seed_r7_navigation)/launch/static_map_navigation.launch">
    <arg name="map_localization_file" value="$(arg map_localization_file)"/>
    <arg name="map_keepout_file"      value="$(arg map_keepout_file)" />
    <arg name="cmd_vel_topic" value="$(arg cmd_vel_topic)"/>
  </include>

</launch>
